src/components/ModalWithForm/ModalWithForm.js

```diff
@@ -11,6 +12,7 @@ function ModalWithForm({
   isActive,
   handleRedirect,
   isFormValid,
+  apiError,
 }) {
   useEscape(handleCloseModal);
 
```

Comment:
```js

document.querySelector('.modal').classList.add('modal_opened');
```
and 

```js

 document.querySelector('.modal').classList.remove('modal_opened');
```

Please, delete it from your project. It's not a react way of closing or opening

You have `closeModal` and `activeModal, setActiveModal` for this logcis